docs/prerequisites/devices.md

```diff
@@ -35,6 +35,7 @@ OpenMQTTGateway is able to scan all the BLE devices that advertise their data so
 | XIAOMI Mi Scale v1 *|XMTZC04HM|weight|
 | XIAOMI Mi Scale v2 *|XMTZC05HM|weight|
 | XIAOMI Mi band * ||steps|
```

Comment:
Why Xiaomi Mi band is not supported by HM10 (asterisk)? I don't have HM10 to test my integration, but if it reads advertisement like ESP32, it should work.

Reply:
The devices that rely on `manufacturerdata` are not supported by HM10, so the Mi band and iNodeEM are not currently supported by HM10.
The gateway extract only `servicedata` with HM10

Reply:
Ok. I will correct docs.